lib/commands/exam.js

```diff
@@ -332,6 +334,44 @@ module.exports = TestCommand.extend({
     }
   },
 
+  /**
+   * Adds an object which maps a list of module from browser id to config
+   * 
+   * @param {object} commandOptions
+   * @param {object} config
+   */
+  _addBrowserModuleMappingToConfig: function(commandOptions) {
+    const filePath = commandOptions.replayExecution;
+    const replayBrowser = commandOptions.replayBrowser;
+
+    let testMappingObj = {}
+
+    try {
+      // Read the replay execution json file.
+      const testMappingRawData = fs.readFileSync(filePath);
+      testMappingObj = JSON.parse(testMappingRawData);
+    } catch (err) { 
+      throw new Error(err);
+    }
+
+    const browserModuleMapping = ((replayBrowser, testMappingObj) => {
+      const browserModuleMappingObj = {};
+      for (const idx in replayBrowser) {
+        const browserId = replayBrowser[idx];
+        const listOfModule = testMappingObj[browserId];
+
+        if(!listOfModule) {
+          throw Error ('browser id doesn\t exist in the file.');
```

Comment:
ID, not id. Also, the error message needs to be more clear about what it means, like 'the file' should really say which file it's talking about, and 'browser id', should be replaced with something saying what the ID is you're looking for (presumably it was specified by the caller?)